package.json

```diff
@@ -15,7 +18,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest",
```

Comment:
I recommend you to use `react-scripts test` here because there is already the jest configuration using the react-scripts